src/components/pipelines-metrics/PipelinesAverageDuration.tsx

```diff
@@ -36,6 +39,42 @@ interface PipelinesAverageDurationProps {
 }
 type DomainType = { x?: DomainTuple; y?: DomainTuple };
 
+const getChartData = (tickValues: number[]|Date[], data: SummaryResponse , type: number) => {
+  const chartData = tickValues?.map((value) => {       
+    const s = data?.summary.find((d) => {
+      if (type == 1){
+        return (
```

Comment:
Instead of values 1, 2, 3 .. for type, can you specify proper names so that it will be easy to understand later as well

Requesting to make this change in all places